src/GdbServer.cc

```diff
@@ -1468,12 +1468,13 @@ static unique_ptr<GdbConnection> await_connection(
   return dbg;
 }
 
-static void print_debugger_launch_command(Task* t, unsigned short port,
+static void print_debugger_launch_command(Task* t, const char* host,
```

Comment:
Make `host` a `const string&`.

Reply:
This is reasonable, I'll change the `push_target_remote_cmd(...)` definition I modified as well.